ebean-migration/src/main/java/io/ebean/migration/runner/LocalMigrationResources.java

```diff
@@ -44,42 +44,68 @@ boolean readInitResources() {
   }
 
   /**
-   * Read all the migration resources (SQL scripts) returning true if there are versions.
+   * Read all the migration resources (SQL scripts and JDBC migrations) returning true if there are versions.
    */
   boolean readResources() {
     if (readFromIndex()) {
       // automatically enable earlyChecksumMode when using index file with pre-computed checksums
       migrationConfig.setEarlyChecksumMode(true);
-      return true;
```

Comment:
I've changed the semantic of "readFromIndex" here.
It returns true, if there is an index file, although if it is empty - so do not fall back to class path scan. We can still have java migrations